src/components/Questions/Header.vue

```diff
@@ -34,13 +43,42 @@ export default defineComponent({
       type: Boolean,
       default: false,
     },
+    /** whether the quiz has a time limit */
+    hasTimeLimit: {
+      type: Boolean,
+      default: false
+    },
+    /** display a warning when <default> minutes left on timer */
+    warningTimeLimit: {
+      type: Number,
+      default: 10
+    },
```

Comment:
In the quiz I tested, a warning was shown when 3 minutes were left
Is that specified somewhere else?

Reply:
This should be in the quiz schema. Just add it as an optional field there no and set a default of 20% of the time limit or something if it is not set there (no need to do migration and all)